Sources/App/Controllers/API/API+PackageController+GetRoute+Model.swift

```diff
@@ -370,32 +355,11 @@ extension API.PackageController.GetRoute.Model {
     }
     
     enum ForkedFromInfo: Codable, Equatable {
-        case fromSPI(
-            packageName: String,
-            originalOwner: String,
-            originalOwnerName: String,
-            originalRepo: String,
-            originalPackageName: String
-        )
+        case fromSPI(originalOwner: String,
+                     originalOwnerName: String,
+                     originalRepo: String,
+                     originalPackageName: String)
         case fromGitHub(url: String)
-        
-        var url: String {
-            switch self {
-            case .fromSPI(_, let originalOwner, _, let originalRepo, _):
-                return SiteURL.package(.value(originalOwner), .value(originalRepo), nil).relativeURL()
-            case .fromGitHub(let url):
-                return url
-            }
-        }
-
-        var ownerURL: String? {
-            switch self {
-            case .fromSPI(_, let owner, _, _, _):
-                return SiteURL.author(.value(owner)).relativeURL()
-            case .fromGitHub:
-                return nil
-            }
-        }
```

Comment:
I've moved these two view related properties into an extension in the file under `Views/PackageController/GetRoute.Model+ext.swift`. That way, view related things don't distract from the basic definition and appear "closer" to where they're actually used.